docs/user-developer-guide.md

```diff
@@ -0,0 +1,8 @@
+# User guide
+Firstly, make sure thar your Android version 8.0 (Oreo) or higher.Then install .apk from [releases](https://github.com/Sairsey/manga-reader/releases).
```

Comment:
version IS - missing verb

It's better to refer to "the latest release" instead of "releases"